Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml.cs

```diff
@@ -113,5 +120,33 @@ private void xBrowseImageLoadButton_Click(object sender, RoutedEventArgs e)
                 }
             }
         }
+
+        private void XBusinessFlowControl_ElementChangedPageEvent()
+        {            
+            if (xBusinessFlowControl != null && xBusinessFlowControl.BusinessFlow != null)
+            {
+                mWizard.mPomLearnUtils.POM.MappedBusinessFlow = xBusinessFlowControl.BusinessFlowRepositoryKey;
+            }
+        }
+
+        private void xPageUrlRadioBtn_Checked(object sender, RoutedEventArgs e)
+        {
+            SetControlsVisibility(false);
+        }
+
+        private void xBusinessFlowRadioBtn_Checked(object sender, RoutedEventArgs e)
+        {
+            SetControlsVisibility(true);
+        }
+
+        private void SetControlsVisibility(bool isBusinessFlow)
+        {
+            if (xPageUrlStackPanel != null && xBusinessFlowControl != null)
+            {
+                mWizard.mPomLearnUtils.POM.PageLoadFlow = ApplicationPOMModel.ePageLoadFlowType.PageURL;
```

Comment:
1- from code it seems that always you setting the flowType as URL
2- we do not use ? : in Ginger code- it is not readable please change it to regular IF statment